sklearn/neighbors/graph.py

```diff
@@ -77,11 +103,13 @@ def kneighbors_graph(X, n_neighbors, mode='connectivity', metric='minkowski',
             ).fit(X)
     else:
         _check_params(X, metric, p, metric_params)
-    return X.kneighbors_graph(X._fit_X, n_neighbors, mode=mode)
+
+    query = _query_include_self(X, include_self, mode)
+    return X.kneighbors_graph(X=query, n_neighbors=n_neighbors, mode=mode)
 
 
-def radius_neighbors_graph(X, radius, mode='connectivity', metric='minkowski',
-                           p=2, metric_params=None):
+def radius_neighbors_graph(X, radius, include_self=None, mode='connectivity',
```

Comment:
I would always add new parameters to the end, in case anyone was relying on parameter order.